stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -10,40 +10,86 @@ This implementation supports introducing all WebSub components: subscribers, pub
 is updated by the topic's publisher.
  
  
-#### Basic Flow with WebSub
+### Basic Flow with WebSub
 1. Subscriber discovers the topics it needs to subscribe to in order to receive updates/content and discovers the hub(s) where it can subscribe.
 
 2. Subscriber sends a subscription request to a hub, specifying the topic it needs to receive notifications for along 
  with other subscription parameters, such as:
-      - The callback URL where content is expected to be delivered
-      - (Optional) A lease seconds value indicating how long the subscriber wants the subscription to stay active
-      - (Optional) A secret to use for [authenticated content distribution](https://www.w3.org/TR/websub/#signing-content)
+    - The callback URL where content is expected to be delivered
+    - (Optional) A lease seconds value indicating how long the subscriber wants the subscription to stay active
+    - (Optional) A secret to use for [authenticated content distribution](https://www.w3.org/TR/websub/#signing-content)
   
-3. The hub sends an intent verification request to the specified callback URL, and if the response indicates verification
+3. The hub sends an intent verification request to the specified callback URL, and if the response indicates 
+verification
```

Comment:
```suggestion
a verification
```

Reply:
Do we need "a" here? It's either verification or denial, and there is one way to verify, which is by echoing the challenge.